### ISSUE TITLE
Warn on runtime-only build

### ISSUE THREAD

**Issue by piduchesne** [jofftiquez/vue-stripe-checkout#31]
Hello, 

Since my update from 1.0.X to 1.1.X i have this warning in console.

Can you help me please ? 

```
[Vue warn]: You are using the runtime-only build of Vue where the template compiler is not available. Either pre-compile the templates into render functions, or use the compiler-included build.

found in

---> <VueStripeCheckout
```>

**Comment by johndatserakis** [comment]
Same here - version 1.1.4.

```javascript
[Vue warn]: You are using the runtime-only build of Vue where the template compiler is not available. Either pre-compile the templates into render functions, or use the compiler-included build.

found in

---> <VueStripeCheckout>
       <Home> at src/components/Layouts/Program/Home.vue
         <App> at src/App.vue
           <Root>
```

**Comment by SMen1** [comment]
I am also getting the above error, with the latest version as I implement for the first time.
I am also getting [Vue warn]: Invalid handler for event "done": got undefined (and same with events "open", "closed", "click") which maybe the other errors referenced in the above pull request requiring work-arounds. If not, if anyone could advise I would be greatful.

**Comment by piduchesne** [comment]
@jofftiquez : i've upgraded from 1.1.4 to 1.1.6 in order to clean the warnings.

Now, i get an error 

`vue.runtime.esm.js?2b0e:1737 ReferenceError: StripeCheckout is not defined
    at VueComponent.setCheckout (index.js?bce0:173)
    at VueComponent.mounted (index.js?bce0:139)
    at callHook (vue.runtime.esm.js?2b0e:2917)
    at Object.insert (vue.runtime.esm.js?2b0e:4154)
    at invokeInsertHook (vue.runtime.esm.js?2b0e:5956)
    at VueComponent.patch [as __patch__] (vue.runtime.esm.js?2b0e:6175)
    at VueComponent.Vue._update (vue.runtime.esm.js?2b0e:2666)
    at VueComponent.updateComponent (vue.runtime.esm.js?2b0e:2784)
    at Watcher.get (vue.runtime.esm.js?2b0e:3138)
    at Watcher.run (vue.runtime.esm.js?2b0e:3215)`

this line causes the error 
`this.checkout = StripeCheckout.configure({ key: this.key });`


Can you help me ?
